pkg/power/platform/source/redfish_util.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package source
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+func getRedfishModel(access RedfishAccessInfo, endpoint string, model interface{}) error {
+	username := access.Username
+	password := access.Password
+	host := access.Host
+
+	// Create a HTTP client and set up the basic authentication header
+	client := &http.Client{}
```

Comment:
Doesn't it required to set `TLSClientConfig` to `&tls.Config{InsecureSkipVerify: true}`?

Reply:
It is better to set `InsecureSkipVerify` as an option (e.g. environment variable) because it is important to secure access.

Reply:
@tiwatsuka @YaSuenag sounds good. can you suggest a change and I'll merge it here. thanks